docs/en/index.md

```diff
@@ -23,3 +23,13 @@ Note that in edge case scenarios, for example if your application wants to modif
 
 > [!WARNING]
 > This module is not compatible with [`silverstripe/hybridsessions`](https://packagist.org/packages/silverstripe/hybridsessions) or [`silverstripe/dynamodb`](https://packagist.org/packages/silverstripe/dynamodb).
+
+## Installation
+
+```bash
+composer require silverstripe/non-blocking-sessions
+```
+
+## GitHub repository
+
+<https://github.com/silverstripe/silverstripe-non-blocking-sessions>
```

Comment:
Replace `Make sure that your Silverstripe CMS installation has [`silverstripe/non-blocking-sessions`](https://github.com/silverstripe/silverstripe-non-blocking-sessions/) installed.` at the top of the document with this

Reply:
Updated